sdk/spring-boms/spring-cloud-azure-dependencies-3/pom.xml

```diff
@@ -0,0 +1,312 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.azure.spring</groupId>
+  <artifactId>spring-cloud-azure-dependencies</artifactId>
+  <version>4.4.0-beta.1</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-dependencies;current} -->
```

Comment:
version not correct

Reply:
all the versions will be updated in the test pr.

Reply:
What did you mean by "all the versions will be updated in the test pr"?

Reply:
It means the versions within the spring-cloud-azure-dependencies-3 will be updated in this [PR](https://github.com/Azure/azure-sdk-for-java/pull/30067).